psalm-baseline.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.19.0@a2ad69ae4f5ab1f7d225a8dc4e2ec2d9415ed599">
+    <file src="src/Action/LoginAction.php">
+        <UndefinedInterfaceMethod occurrences="1">
+            <code>getFlashBag</code>
```

Comment:
Soon, it won't be needed https://github.com/psalm/psalm-plugin-symfony/pull/242 ;)